pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -2089,3 +2114,59 @@ func getRouteHostname(obj interface{}) (host string) {
 	}
 	return host
 }
+
+func checkAndCreateAdminSecret(service kubernetes.PlatformService, namespace string, secretName string, environment api.EnvironmentType) (error, string, string) {
+	customerSecret, err := getSecret(service, namespace, secretName)
+
+	// The secret passed by the user doesn't exist we create one with default values
+	if err != nil {
```

Comment:
is this check correct? If secret exists, then is return err ??

Reply:
If secret exists the err is nil